guiEditor/src/components/parentingPropertyGridComponent.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from "react";
+import { OptionsLineComponent } from "../sharedUiComponents/lines/optionsLineComponent";
+import { LineContainerComponent } from "../sharedUiComponents/lines/lineContainerComponent";
+import { GlobalState } from "../globalState";
+import { Control } from "babylonjs-gui/2D/controls/control";
+import { Container } from "babylonjs-gui/2D/controls/container";
+
+interface IParentingPropertyGridComponentProps {
+    guiNode: Control;
+    guiNodes: Control[];
+    globalState: GlobalState;
+}
+
+export class ParentingPropertyGridComponent extends React.Component<IParentingPropertyGridComponentProps> {
+    constructor(props: IParentingPropertyGridComponentProps) {
+        super(props);
+    }
+    public parentIndex: number = 0;
+
+    private _isContainer(guiControl: Control) {

```

Comment:
Would it make sense having it in the main repository?

Reply:
I ad it there originally but I didn't see it being used any where but here for now.